tests/function_corpus.py

```diff
@@ -990,6 +994,11 @@ def tensor_courpus_max_cos(x, y):
     TestFunc(tensor_courpus_broadcast, broadcast_strategy(), static_argnames={'shape'}),
 ]
 
+basic_tensor_tests.extend([
+    TestFunc(tensor_courpus_jacobian_add, elementwise_strategy(2)),
+    TestFunc(tensor_courpus_jacobian_mul, elementwise_strategy(2))
+])
```

Comment:
These are already in the list with broadcasting. It's fine to add a second even simpler list but I don't think you should be adding repeats here.